src/main/java/pl/reservo/reservoservice/model/ApplicationUser.java

```diff
@@ -0,0 +1,28 @@
+package pl.reservo.reservoservice.model;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import pl.reservo.reservoservice.constants.UserRole;
+
+
+@Entity
+@Data
```

Comment:
`@Data` not recomended on entities

Reply:
Fixed but I am not sure if in correct way